test/fixtures/requests/getpaths/invalid.json

```diff
@@ -1,16 +0,0 @@
-{
```

Comment:
it's unnecessary to remove these right now - we've got another ticket for dealing with unused fixtures